config/config.go

```diff
@@ -39,20 +39,22 @@ type ServicesConfig struct {
 }
 
 type SidecarConfig struct {
-	ExcludeIPs           []string      `envconfig:"EXCLUDE_IPS" default:"192.168.168.168"`
-	Discovery            []string      `envconfig:"DISCOVERY" default:"docker"`
-	StatsAddr            string        `envconfig:"STATS_ADDR"`
-	PushPullInterval     time.Duration `envconfig:"PUSH_PULL_INTERVAL" default:"20s"`
-	GossipMessages       int           `envconfig:"GOSSIP_MESSAGES" default:"15"`
-	GossipInterval       time.Duration `envconfig:"GOSSIP_INTERVAL" default:"200ms"`
-	HandoffQueueDepth    int           `envconfig:"HANDOFF_QUEUE_DEPTH" default:"1024"`
-	LoggingFormat        string        `envconfig:"LOGGING_FORMAT"`
-	LoggingLevel         string        `envconfig:"LOGGING_LEVEL" default:"info"`
-	DefaultCheckEndpoint string        `envconfig:"DEFAULT_CHECK_ENDPOINT" default:"/version"`
-	Seeds                []string      `envconfig:"SEEDS"`
-	ClusterName          string        `envconfig:"CLUSTER_NAME" default:"default"`
-	AdvertiseIP          string        `envconfig:"ADVERTISE_IP"`
-	BindPort             int           `envconfig:"BIND_PORT" default:"7946"`
+	ExcludeIPs             []string      `envconfig:"EXCLUDE_IPS" default:"192.168.168.168"`
+	Discovery              []string      `envconfig:"DISCOVERY" default:"docker"`
+	StatsAddr              string        `envconfig:"STATS_ADDR"`
+	PushPullInterval       time.Duration `envconfig:"PUSH_PULL_INTERVAL" default:"20s"`
+	GossipMessages         int           `envconfig:"GOSSIP_MESSAGES" default:"15"`
+	GossipInterval         time.Duration `envconfig:"GOSSIP_INTERVAL" default:"200ms"`
+	HandoffQueueDepth      int           `envconfig:"HANDOFF_QUEUE_DEPTH" default:"1024"`
+	LoggingFormat          string        `envconfig:"LOGGING_FORMAT"`
+	LoggingLevel           string        `envconfig:"LOGGING_LEVEL" default:"info"`
+	DefaultCheckEndpoint   string        `envconfig:"DEFAULT_CHECK_ENDPOINT" default:"/version"`
+	Seeds                  []string      `envconfig:"SEEDS"`
+	ClusterName            string        `envconfig:"CLUSTER_NAME" default:"default"`
+	AdvertiseIP            string        `envconfig:"ADVERTISE_IP"`
+	BindPort               int           `envconfig:"BIND_PORT" default:"7946"`
+	Debug                  bool          `envconfig:"DEBUG" default:"false"`
+	DiscoverySleepInterval time.Duration `envconfig:"DISCOVERY_SLEEP_INTERVAL" default:"1s"`
```

Comment:
Make Sleep interval configurable. This is useful because running `kubectl` every second is a little extreme